etc/ci/upload_nightly.sh

```diff
@@ -16,25 +16,61 @@ usage() {
 
 
 upload() {
-    local nightly_filename nightly_timestamp
-    nightly_timestamp="$(date -u +"%Y-%m-%dT%H-%M-%SZ")"
-    nightly_filename="${nightly_timestamp}-$(basename "${2}")"
+    local nightly_filename
+    nightly_filename="${4}-$(basename "${2}")"
     local -r nightly_upload_dir="s3://servo-builds/nightly/${1}"
     local -r package_upload_path="${nightly_upload_dir}/${nightly_filename}"
     s3cmd --mime-type="application/octet-stream" \
           put "${2}" "${package_upload_path}"
     s3cmd cp "${package_upload_path}" "${nightly_upload_dir}/servo-latest.${3}"
 }
 
+update_brew() {
+  echo "Updating brew formula"
+
+  local package_url sha version script_dir tmp_dir nightly_filename
+
+  nightly_filename="${2}-$(basename "${1}")"
+  package_url="https://download.servo.org/nightly/macbrew/${nightly_filename}"
+  sha="$(shasum -a 256 ${1} | sed -e 's/ .*//')"
+
+  # This will transform a timestamp (2016-12-13T08-01-10Z for example)
+  # into a valid brew version number (2016.12.13).
+  version="$(echo "${2}" | \
+    sed -n 's/\([0-9]\{4\}\)-\([0-9]\{2\}\)-\([0-9]\{2\}\).*/\1.\2.\3/p')"
+
+  script_dir="${PWD}/$(dirname "${0}")"
+  tmp_dir="$(mktemp -d -t homebrew-servo)"
```

Comment:
Oh, I didn't know that, that's a nice feature. Still might be nice to have in case this gets blindly copy-pasted out of context later, but up to you if you want to change it.